src/main/java/com/salesforce/einsteinbot/sdk/client/BasicChatbotClientImpl.java

```diff
@@ -86,11 +96,34 @@ void setHealthApi(HealthApi healthApi) {
     this.healthApi = healthApi;
   }
 
+  @VisibleForTesting
+  void setVersionsApi(VersionsApi versionsApi) {
+    this.versionsApi = versionsApi;
+  }
+
   @Override
   public BotResponse startChatSession(RequestConfig config,
       ExternalSessionId sessionId,
       BotSendMessageRequest botSendMessageRequest) {
 
+    String currentApiVersion;
+    try {
+      currentApiVersion = getCurrentApiVersion();
+      SupportedVersions versions = getSupportedVersions();
+      List<SupportedVersionsVersions> supportedVersions = versions.getVersions()
+          .stream()
+          .filter(v -> {
+            assert v.getVersionNumber() != null;
```

Comment:
Assertions can be disabled using `-disableassertions` jvm flag. It's not recommended to use assertions.

Reply:
Refactored this to not have `assert`